R/AbstractPackageReporter.R

```diff
@@ -34,9 +39,10 @@ AbstractPackageReporter <- R6::R6Class(
     
     public = list(
         
-        set_package = function(pkg_name, pkg_path = NULL) {
+        set_package = function(pkg_name, pkg_path = NULL, pkg_lib = .libPaths()[1]) {
```

Comment:
I'm still confused by this. 99.99999999% of users will not need to customize their library. Are we only doing this for the CRAN testing thing?

I would rather have us manually override `libPaths` at test time or something